### PR TITLE
Feature/testcases/filter/apiintegration

### PR DESCRIPTION
**Pull request by AsmitaMargaje**
resolved all defects

### CODE REVIEW

File: src/screens/TestCases/Modal/CustomFilterModal.jsx [hunk 1]
```diff
@@ -20,7 +20,11 @@ const CustomFilterModal = ({
   type,
   handleApplyButton,
   filterData,
-  localDispatch
+  localDispatch,
+  handleClearAllButton,
+  handleSearchChange,
+  handleClearAllFilter,
```
Comment (AmitSolnke) [inline src/screens/TestCases/Modal/CustomFilterModal.jsx:26]: Not used than remove it.
Avoid rebundancy

File: src/redux/services/testCases/testCaseReview/index.js
Comment (AmitSolnke) [inline src/redux/services/testCases/testCaseReview/index.js:1]: For Mobile view,
Make all button col-12

File: src/screens/TestCases/Modal/CustomFilterModal.jsx
Comment (AmitSolnke) [inline src/screens/TestCases/Modal/CustomFilterModal.jsx:1]: There is no use of useEffect  ,
I assume it has added for resize  the modal but resizing can be handled with grid in bootstrap.
This is not a appropiate way

File: src/screens/TestCases/Modal/CustomFilterModal.jsx (hunk 1 above)
Comment (AsmitaMargaje) [reply src/screens/TestCases/Modal/CustomFilterModal.jsx:25]: resolved

File: src/redux/services/testCases/testCaseReview/index.js
Comment (AsmitaMargaje) [reply src/redux/services/testCases/testCaseReview/index.js:1]: resolved

File: src/screens/TestCases/Modal/CustomFilterModal.jsx
Comment (AsmitaMargaje) [reply src/screens/TestCases/Modal/CustomFilterModal.jsx:1]: in next phase i will work on it

**Pull request merged**